File: photoeditor/src/test/java/ja/burhanrashid52/photoeditor/EnumTest.java
package ja.burhanrashid52.photoeditor;

import org.junit.Test;

import ja.burhanrashid52.photoeditor.shape.ShapeType;

import static org.junit.Assert.assertEquals;

public class EnumTest {

    @Test
    public void testNumberOfViewTypes() {
        assertEquals(ViewType.values().length, 4);
    }

    @Test
    public void testNumberOfShapeTypes() {
        assertEquals(ShapeType.values().length, 4);
    }

    @Test
    public void testNumberOfPhotoFilterTypes() {
        assertEquals(PhotoFilter.values().length, 24);
    }

}

File: photoeditor/src/test/java/ja/burhanrashid52/photoeditor/EnumTest.kt
package ja.mhdxbilal007.photoeditor

import ja.mhdxbilal007.photoeditor.shape.ShapeType
import junit.framework.TestCase.assertEquals
import org.junit.Assert
import org.junit.Test

class EnumTest {
    @Test
    fun testNumberOfViewTypes() {
        assertEquals(ViewType.values().size.toLong(), 4)
    }

    @Test
    fun testNumberOfPhotoFilterTypes() {
        assertEquals(PhotoFilter.values().size.toLong(), 24)
    }
}